manage-apps/register.html

```diff
@@ -206,6 +206,10 @@
           $('#sandboxRegistration button[type="submit"]').attr('disabled', true);
         },
         error: function(res) {
+          console.log('******',res)
+          if (res.statusText === 'error') {
```

Comment:
Test went through and it looks like this change should address potential double error messages.